lockfile.yml

```diff
@@ -37,10 +42,13 @@ packages:
     repo: syberia/tundra
   -
     name: syberia
-    version: 0.6.1.9006
+    version: 0.6.1.9007
     repo: syberia/syberia
     ref: 0.6-dev
   -
     name: mungebits2
     version: 0.1.0.9002
     repo: syberia/mungebits2
+  -
+    name: gbm
+    version: 2.1.1
```

Comment:
CRAN and latest github version differ significantly in terms of features, make sure you use the right one